drivers/include/cc110x.h

```diff
@@ -574,7 +582,7 @@ int cc110x_setup(cc110x_t *dev, const cc110x_params_t *params);
  * @ref cc110x_full_calibration is called to update it.
  */
 int cc110x_apply_config(cc110x_t *dev, const cc110x_config_t *conf,
-                        const cc110x_chanmap_t *chanmap);
+                        const cc110x_chanmap_t *chanmap, uint8_t channel);
```

Comment:
> `The following parameters of cc110x_apply_config(cc110x_t *dev, const cc110x_config_t *conf, const cc110x_chanmap_t *chanmap, uint8_t channel) are not documented: channel`

Reply:
I think I fucked up merging. Thanks for noticing!